saleor/graphql/account/mutations/permission_group.py

```diff
@@ -444,3 +449,12 @@ def check_if_group_can_be_removed(cls, group):
             raise ValidationError(
                 {"id": ValidationError(message=msg, code=code, params=params)}
             )
+
+    @classmethod
+    def ensure_not_removing_requestor_last_group(cls, group, requestor):
+        """Ensure user doesn't remove user's last group."""
+        if requestor in group.user_set.all() and requestor.groups.count() == 1:
+            # add error
```

Comment:
This comment is unclear for me. I see that you are rising a error instead of adding it